using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -22,64 +25,73 @@ public static void syncIdentifier(final String identifierType,
                                       final VisitorID.AuthenticationState authenticationState);
 ```
 
+o  *identifierType* (String) containing identifier type, and the *identifier* (String) containing identifier value; should not be null or empty. If the identifierTtype and identifier do not have valid string values \(non null or empty\), that identifier is ignored by the Identity extension.
+
+o  *authenticationState* indicates the authentication state of the user, which contains one of the VisitorID.AuthenticationState values `VisitorID.AuthenticationState.AUTHENTICATED`, `VisitorID.AuthenticationState.LOGGED_OUT`, `VisitorID.AuthenticationState.UNKNOWN`
+
 #### **Example**
 
 ```java
-Identity.syncIdentifier("idType", "idValue", VisitorID.AuthenticationState.AUTHENTICATED);
+Identity.syncIdentifier("idType", 
+                        "idValue", 
+                        VisitorID.AuthenticationState.AUTHENTICATED);
 ```
 
-### syncIdentifiers
+### syncIdentifiers<a id="syncIdentifiers-java"></a>
 
-{% hint style="info" %}
-The `identifiers` map contains IDs with the Identifier type as the key, and the string identifier as the value. If the identifier type and identifier do not have valid string values \(non null or empty\), the identifier is ignored by the Identity extension.
-{% endhint %}
+The function of the `syncIdentifiers`  API is same as the `syncIdentifier` API, which passes a list of identifiers each containing an identifier type as the key and an identifier as the value. In each of the identifiers pair, both identifier type and identifier should be non empty and non null values, otherwise they will be ignore. 
```

Comment:
- Please replace this text: "list of identifiers each containing" with "list of identifiers, and each identifier contains"
- Please reword the 2nd sentence so that it reads as follows:
"In each identifier pair, if the the identifier type and identifier are empty or have non null values, these values are ignored."